openslides/core/signals.py

```diff
@@ -20,3 +22,56 @@ def delete_django_app_permissions(sender, **kwargs):
         Q(app_label='sessions'))
     for permission in Permission.objects.filter(content_type__in=contenttypes):
         permission.delete()
+
+
+def create_builtin_projection_defaults(**kwargs):
+    """
+    Creates the builtin defaults:
+     - agenda_all_items, agenda_item
+     - assignments
+     - mediafiles
+     - motion
+     - users
+     - list_of_speakers
+     - current_list_of_speakers
+    """
+
+    # Check whether the pk's from 1 to 8 are free.
+    if ProjectionDefault.objects.filter(pk__in=range(1, 9)).exists():
```

Comment:
ProjectionDefault.objectsProjectionDefault.objects.all(). It should do nothing if at least one element exists.